src/strings/csv/en.csv

```diff
@@ -1476,6 +1477,8 @@ PROPOSED_RESTORATION_ACTIVITIES,Proposed Restoration Activities,
 PROVIDE_PLANT_DENSITY_PER_SPECIES,Provide plant density per species,
 PROVIDE_PLANT_DENSITY_PER_SPECIES_TOOLTIP,Enter the densities for each species planted. The plant densities based on the Withdrawal Log data are shown and can be entered as the plant densities.,
 PUBLISH,Publish,
+PUBLISH_ACTIVITY,Publish Activity,
+PUBLISH_ACTIVITY_MODAL_MESSAGE,You are about to publish Planting from {0}. Are you sure?,
```

Comment:
Is this message always supposed to say "Planting"? Or is it supposed to be something like this:
```
You are about to publish {0} from {1}. Are you sure?
```
That way we could pass in the activity type?